app/helpers/cloud_network_helper/textual_summary.rb

```diff
@@ -58,4 +58,8 @@ def textual_network_routers
   def textual_cloud_subnets
     @record.cloud_subnets
   end
+
+  def textual_floating_ips
+    textual_link(@record.floating_ips, :label => _("Floating Ips"))
```

Comment:
I think in most places we use `Floating IPs` rather than `Floating Ips` (those places with `Floating Ips` should be fixed IMO).